.github/workflows/ironclaw-release.yml

```
@@ -251,6 +251,75 @@ jobs:
+    needs:
+      - plan
+      - build-local-artifacts
+      - build-global-artifacts
+    if: ${{ always() && needs.plan.result == 'success' && needs.plan.outputs.publishing == 'true' && needs.build-local-artifacts.result == 'success' && needs.build-global-artifacts.result == 'success' }}
+    runs-on: ubuntu-22.04
+    timeout-minutes: 20
+    env:
+      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      PREVIOUS_RELEASE_TAG: ironclaw-v1.0.0-rc.1
+      CANARY_TARGET: x86_64-unknown-linux-gnu
+      CANARY_ARTIFACT_DIR: ${{ runner.temp }}/release-upgrade-canary
```

actionlint (1.7.12)

[error] context "runner" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)